ocs_ci/ocs/platform_nodes.py

```diff
@@ -73,12 +73,12 @@ def restart_nodes(self, nodes, force=True):
             "Restart nodes functionality is not implemented"
         )
 
-    def detach_volume(self, node):
+    def detach_volume(self, volume, node):
```

Comment:
Applies to all function's signature changes below, please check that no usage will cause backward compatibility failures

Reply:
ack. Will change